agent/selfupdater/updater_docker.go

```diff
@@ -112,6 +112,21 @@ func (d *dockerUpdater) CompleteUpdate() error {
 			}
 		}
 
+		// Append /etc/resolv.conf to mount if old container
+		// version is less than v0..0 since /etc/resolv.conf
+		// is required inside the container to update host
```

Comment:
Here the comment is still wrong.

Reply:
Ahh, been a while since I've worked on a project where we don't just squash and merge PRs. Should be fixed in all commits that include the change.

Reply:
We try to keep the commit history as clean as possible as it helps in the future. Great job.